app/frontend/javascript/labware-custom-metadata/components/LabwareCustomMetadataAddForm.spec.js

```diff
@@ -295,8 +295,8 @@ describe('LabwareCustomMetadataAddForm', () => {
       })
     })
 
-    describe('#buildPayload', () => {
-      // todo
-    })
+    // TODO: add this test suite
```

Comment:
Is this something that will be done imminently?  If not, can you add a story to the backlog and put the identifier in this TODO?